### PR TITLE
todo app deployment to s3

### PR DESCRIPTION
**Pull request by samalba**
Implements todoapp deployment to s3 static website

### CODE REVIEW

File: todoapp/.dagger/env/staging/plan/main.cue [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+    "dagger.io/dagger"
```
Comment (aluzzardi) [inline todoapp/.dagger/env/staging/plan/main.cue:4]: `make cuefmt`

File: todoapp/.dagger/env/staging/plan/main.cue [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+// Build the source code using Yarn
+app: yarn.#Package & {
+	"source": source
+}
+
+// Host the application on an S3 bucket
+s3bucket: s3.#Sync & {
+	source: app.build
+	target: "s3://todo.microstaging.io/\(appName)/"
```
Comment (aluzzardi) [inline todoapp/.dagger/env/staging/plan/main.cue:20]: should the bucket be an input as well? If we use the same config for cloudformation as well

**Review by aluzzardi — COMMENTED** [review]
Should we call the environment `s3` rather than `staging`?

We'll also have `kubernetes`, `cloudrun` and `cloudformation` environments.

**Comment by samalba** [comment]
SGTM, I'll make those changes

**Comment by aluzzardi** [comment]
LGTM -- CI is failing (markdown linter on the todoapp). We could either fix the issues or remove the README file (we'll probably want to replace it with our own eventually linking to the docs anyway right?)

**Pull request merged**